godot-core/src/meta/mod.rs

```diff
@@ -40,13 +40,17 @@ mod godot_convert;
 mod method_info;
 mod property_info;
 mod ref_arg;
+#[cfg(feature = "codegen-full")]
+mod rpc_config;
```

Comment:
Maybe add a brief comment that `MultiplayerPeer::TransferMode` and `MultiplayerApi::RpcMode` aren't part of the basic set of classes (also below).